gax-grpc/src/test/java/com/google/api/gax/grpc/testing/LocalChannelProvider.java

```diff
@@ -43,10 +43,10 @@
 import io.grpc.ForwardingClientCall.SimpleForwardingClientCall;
 import io.grpc.Metadata;
 import io.grpc.MethodDescriptor;
-import io.grpc.netty.NegotiationType;
-import io.grpc.netty.NettyChannelBuilder;
-import io.netty.channel.local.LocalAddress;
-import io.netty.channel.local.LocalChannel;
+import io.grpc.netty.shaded.io.grpc.netty.NegotiationType;
+import io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder;
+import io.grpc.netty.shaded.io.netty.channel.local.LocalAddress;
+import io.grpc.netty.shaded.io.netty.channel.local.LocalChannel;
```

Comment:
If this is shaded, we need to stop depending on it. We are just using the in-process netty stuff because it was handy; can we find a replacement?